### PR TITLE
Allow to create CRS from other trusted setup

### PR DESCRIPTION
**Pull request by dapplion**
Current CRS class only allow init from random or serialized JSON. Expose also to construct from a list of G1 points

### CODE REVIEW

File: curdleproofs/curdleproofs/crs.py [hunk 1]
```diff
@@ -44,23 +44,22 @@ def __init__(
...
+        vec_H = points[ell:ell+n_blinders]
         return cls(
             vec_G=vec_G,
             vec_H=vec_H,
-            H=H,
-            G_t=G_t,
-            G_u=G_u,
-            G_sum=G_sum,
-            H_sum=H_sum,
+            H=points[ell+n_blinders + 1],
```
Comment (asn-d6) [inline curdleproofs/curdleproofs/crs.py:58]: Wait H == G_t?

Comment (asn-d6) [reply curdleproofs/curdleproofs/crs.py:58]: This logic seems fragile

Comment (asn-d6) [reply curdleproofs/curdleproofs/crs.py:58]:
```
            H=points[ell+n_blinders + 1],
            G_t=points[ell+n_blinders + 1],
```
But you seem to be assigning the same random point to `H` and `G_t`, no?

Comment (dapplion) [reply curdleproofs/curdleproofs/crs.py]: oh ok I'm blind, fixed typo

**Review by asn-d6 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**